tg_bot/modules/log_channel.py

```diff
@@ -0,0 +1,263 @@
+from functools import wraps
```

Comment:
@iflare3g @Kavuti roba vostra :)

Reply:
C'è da fixare qualcosa ?

Reply:
> C'è da fixare qualcosa ?

Non credo, ma non ho avuto modo di esaminare bene il codice, mi riuscite a dare un check che l'integrazione con i nuovi moduli sia ok e non ci siano problemi di crash nel caso in cui arrivi un link malformato?

Reply:
@Kavuti ci ha lavorato di più e meglio di me

Reply:
Ho effettuato diversi test giorni addietro e sembrava tutto ok, giusto prima di iniziare la review. Se volete possiamo comunque pensare di fare un'altra sessione di test prima di deployare... ditemi voi.